encoding/asn1/asn1.go

```diff
@@ -468,10 +468,12 @@ func isPrintable(b byte, asterisk asteriskFlag, ampersand ampersandFlag) bool {
 // parseIA5String parses an ASN.1 IA5String (ASCII string) from the given
 // byte slice and returns it.
 func parseIA5String(bytes []byte) (ret string, err error) {
-	for _, b := range bytes {
-		if b >= utf8.RuneSelf {
-			err = SyntaxError{"IA5String contains invalid character"}
-			return
+	if !AllowPermissiveParsing {
```

Comment:
Am I correct in assuming these changes in `asn1.go` are unrelated to the PR description?

Reply:
it was a pending previous PR

Reply:
updated commit info